core/src/std_packages.rs

```diff
@@ -0,0 +1,93 @@
+use std::collections::HashMap;
+
+use crate::context::Either;
+use crate::context::Either::{Left, Right};
+use crate::std_packages_macros::{define_native_packages, define_standard_package_loader};
+use crate::{ArgInfo, Context, CoreError, Element, OutputFormat, PackageInfo, Transform};
+
+// Here, all native packages are declared. The macro expands to two functions,
+// one being a function returning the manifests for those packages, and the other
+// being the entry point to run these packages.
+define_native_packages! {
+    "core" => {
+        "raw", vec![] => native_raw,
+    };
+    "reparse" => {
+        "inline_content", vec![] => native_inline_content,
+        "block_content", vec![] => native_block_content,
+    };
+    "env" => {
+        "set-env", vec![
+            ArgInfo {
+                name: "key".to_string(),
+                default: None,
+                description: "The key to set".to_string()
+            }
+        ] => native_set_env,
+    };
+}
+
+// Here, all standard packages are declared. The macro expands to one function
+// which takes a &mut Context and loads it with the given standard package. It is important that the
+// package with a given name both is in a folder with that name, containing a cargo package with
+// that name. Otherwise, the module won't be found
+define_standard_package_loader! {
```

Comment:
Might be nice to move `define_standard_package_loader` above the native_package macro expressions since most of the time people will add non-native packages and being at the very top makes it a bit easier to find!

Reply:
Fixed, I moved it up